### ISSUE TITLE
Cannot switch between Markdown editor and preview if accessibility VoiceOver feature is active

### ISSUE THREAD

**Issue by iamgabrielma** [Automattic/simplenote-ios#1172]
Reported on 3757989-zen

> Using the IOS built in screen-reader Voiceover, I want to be able to access the preview function for notes which are written in Markdown. On Windows the command is control+shift.p. I read your blog post on Markdown which indicated you could somehow swipe to trigger a preview on the IOS app, but if there's a way to make that work with Voiceover I haven't found it yet. So the easiest I think would to add a "Preview" button.

Details on accessibility options can be found at https://www.apple.com/accessibility/vision/

### Expected
To be able to swipe between the Markdown editor and preview using either two-fingers (like default) or three-fingers (specific when VoiceOver is active).

### Observed
We cannot switch between editor and preview modes.

### Reproduced

1. In your device, go to Settings > Accessibility > VoiceOver, and enable it. Note that this will change the way your phone works.
2. Open the Simplenote app by double tapping (if is on a different sub-screen, you can use three-finger swipe to move through) 
3. Try to switch between the editor and the preview mode in any of your notes, either by two-finger or three-finger swiping, this is not responsive.

The user suggest we could add a simple button if VoiceOver is active in order to switch between modes.

Reproduced on Apple | iPhone7 | iOS 14.2 | Simplenote 4.31
